backend/src/main/java/com/gestionnaire_de_stage/service/StudentService.java

```diff
@@ -117,4 +114,14 @@ private boolean isEmailAndPasswordValid(String email, String password) {
         return studentRepository.existsByEmailAndPassword(email, password);
     }
 
+    public Student getOneByEmail(String email) throws EmailDoesNotExistException {
+        Assert.notNull(email, "Le courriel est null");
+        if (isEmailInvalid(email)) throw new EmailDoesNotExistException();
```

Comment:
```suggestion
        if (isEmailInvalid(email)) 
            throw new EmailDoesNotExistException();
```
C'est seulement dans les langages familier ou le one line if est correct.
Dans java, c'est toujours en plusieurs lignes.